source/clear-linux/guides/network/custom-clear-container.rst

```diff
@@ -0,0 +1,369 @@
+.. _custom-clear-container:
+
+Build a Custom |CL|-based Docker Container Image
```

Comment:
Headings/titles should be in sentence case. The |CL| substitution doesn't work in headings. Let's avoid a hyphen following Clear Linux
- Have only the first letter of: the sentence, Clear Linux, and Docker be capitalized.
- Write out the full "Clear Linux\\*" and drop the hyphen.
- Docker\\* (the \\ ensures the * doesn't make everything enclosed italic)

Build a custom Clear Linux\\* based Docker\\* container image